OMEdit/OMEditGUI/Animation/AbstractAnimationWindow.cpp

```diff
@@ -129,6 +130,11 @@ void AbstractAnimationWindow::openAnimationFile(QString fileName)
         mpPerspectiveDropDownBox->setCurrentIndex(1);
         cameraPositionSide();
       }
+
```

Comment:
Done. The call to stashView in the update function is only called for composite models anyway, so there no boolean is needed.